tailwind.config.cjs

```diff
@@ -56,7 +56,7 @@ module.exports = {
     },
     extend: {
       aspectRatio: {
-        image: '4 / 3',
+        image: '3 / 2',
```

Comment:
add new aspect reatio dont replace with existing one. because if some other person is using this aspect ratio then his/her code will not work